### ISSUE TITLE
Yabai Doesn't Start, "Accessibility Access" Keeps Popping Up

### ISSUE THREAD

**Issue by rjhilgefort** [koekeishiya/yabai#2063]
Since upgrading to Sonoma `14.2.1`, yabai fails to start it seems. It keeps popping up the "Accessibility Access" prompt and despite me checking the box, doesn't seem to take. Yabai doesn't seem to reorg the windows or anything. I don't see any errors in the system console either. Any ideas?
<img width="473" alt="image" src="https://github.com/koekeishiya/yabai/assets/2034823/d7c2feba-0cdc-469e-b250-d09fab3ea738">
<img width="214" alt="image" src="https://github.com/koekeishiya/yabai/assets/2034823/45126d44-4fcb-4a34-8a43-19c4157d7571">
![CleanShot 2024-01-19 at 11 23 42@2x](https://github.com/koekeishiya/yabai/assets/2034823/a2a5fff6-36e6-44ed-8d1b-4cddaf2ebca7)

**Comment by rjhilgefort** [comment]
I see https://github.com/koekeishiya/yabai/issues/1823 and 

> You need to codesign the binary when building from source, as mentioned in the wiki.

but I'm not building from source. Anything else to try?

**Comment by rjhilgefort** [comment]
I also have trouble making sure SIP is disabled. The instructions in the wiki say to do “enable”, but then the prompt doesn’t recognize any users

![IMG_3635](https://github.com/koekeishiya/yabai/assets/2034823/ede35d56-9a18-4763-920b-c214d7fe2b27)

if I use “disable” instead, I get usage error.

![IMG_3634](https://github.com/koekeishiya/yabai/assets/2034823/cf18bec7-00bc-47b2-ba88-f099b3d31933)

(Sorry for not actual screen grabs, no screen capture tools in recovery mode)

**Comment by rjhilgefort** [comment]
False alarm I guess. I uninstalled with `brew uninstall yabai` and then `brew install yabai` and things started working again. 🤷‍♂️
